modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormTextInput.ts

```diff
@@ -17,7 +17,11 @@ export const renderContextFormTextInput = (
   valueState: Singleton.Value<string>
 ): SketchSpec => {
   const editorOffCell = Cell(Fun.noop);
-  const getFormApi = (comp: AlloyComponent) => ContextFormApi.getFormApi<string>(comp, valueState);
+  const getFormParentApi = (comp: AlloyComponent, focusfallbackElement?: SugarElement<HTMLElement>) => {
```

Comment:
Should we move this to the `ContextFormApi` is this an issue with other inputs or just text input can we add tests to those if they are not covered?